### PR TITLE
Dockingstation fixes

### PR DESCRIPTION
**Pull request by aldoram5**
Fixes https://github.com/Monika-After-Story/MonikaModDev/issues/3659, https://github.com/Monika-After-Story/MonikaModDev/issues/3801 and maybe/hopefully all the other issues dockingstation has.

Basically the issue was how we were handling the pickled bytes and adding that to the Monika file, this one encodes the bytes into base64 first and then stores that, when reading it also takes care of that.

## Testing
- Taking Monika out on mac should work
- Taking Monika out on windows should work too

If anyone has a mac and can test this too and give us feedback it'll be greatly appreciated.

### CODE REVIEW

**Comment by Jacobchu025** [comment]
Mac user reporting. Will test presently.

**Comment by Jacobchu025** [comment]
Heya.

File generated fine.

Moved the "Monika" file to my Desktop, then back to "characters" and I just see the empty room.

Sent my "Monika" file and persistent file to @multimokia and will get to you guys back soon.

**Comment by aldoram5** [comment]
Once this one is merged we need to re-enable taking Monika out on macs

**Comment by Jacobchu025** [comment]
Success! Thanks to Aldo and multi.

File generated fine, returned fine, and dialogue was correct.

If you need anything else, let me know.

File: Monika After Story/game/zz_dockingstation.rpy [hunk 1]
```diff
@@ -1983,7 +1987,11 @@ init 200 python in mas_dockstat:
         RETURNS: a persistent object, or None if failure
         """
         try:
-            return cPickle.loads(str(data_line))
+            #pers = re.match(r"^(.*?)\|\|\|per\|",str(data_line)).group()
+            splitted = data_line.split("|||per|")
```
Comment (ThePotatoGuy) [inline Monika After Story/game/zz_dockingstation.rpy]:
```suggestion
            # TODO: change separator to a very large delimeter so we can handle persistents larger than 4MB
            splitted = data_line.split("|||per|")
```

**Pull request merged**